src/components/FarmCard/FarmCard.tsx

```diff
@@ -116,7 +126,12 @@ const FarmCard: React.FC<{
   );
 
   return (
-    <Box className={classes.farmLPCard}>
+    <Box
+      className={cx(
+        classes.farmLPCard,
```

Comment:
Hmm are you resolving class names based off of conditions?

Reply:
Yes